ember-cli/2017-11/november-9.md

```diff
@@ -0,0 +1,8 @@
+# ember-cli Meeting Notes - 2017-11-09
+
+## Attendees
+
+## Agenda
```

Comment:
Please reply here with agenda items.

Reply:
`eslint-plugin-ember` situation

Reply:
F2F @ EmberConf

Reply:
https://github.com/ember-cli/ember-cli/issues/7421

Reply:
* ember-try in ember-cli dependencies https://github.com/ember-cli/ember-cli/issues/7421
* `ember-cli-qunit` -> `ember-qunit` ?